tests/testdata_gcp_kpt/gcp/Kptfile

```diff
@@ -11,37 +11,31 @@ openAPI:
         setter:
           name: gcloud.core.project
           value: customerProject
-          isSet: true
```

Comment:
I ran the hack/generate-xxx script, but looks like this was automatically removed

Reply:
Can you revert the changes? You shouldn't need to rerun the generate script; it was a one time deal.

Reply:
Does that mean I don't need to update the testdata for kpt anymore?